examples/android/Cargo.toml

```diff
@@ -7,23 +7,38 @@ publish = false
 license = "MIT OR Apache-2.0"
 
 [lib]
-name = "bevy_android_example"
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "rlib"]
```

Comment:
not sure what the `rlib` is for here?

Reply:
rlib is for the main.rs, so its easy to run on desktop.  If I remove it change [[bin]] src to lib.rs like you did in android-activity examples, they work but then you get these warnings:
```
 warning: ~/bevy/examples/android/Cargo.toml: file found to be present in multiple build targets: ~/bevy/examples/android/src/lib.rs
```